src/nvim/decoration.c

```diff
@@ -15,6 +16,10 @@
 
 static PMap(uint64_t) hl_decors;
 
+#define DECORATION_WATCHED_INIT(col) { KV_INITIAL_VALUE, KV_INITIAL_VALUE, 0, kVTOverlay, \
```

Comment:
instead of duplicating all this, better to use `DECORATION_INIT` and then set `ui_watched` to true (makes sense anyway if we add a `ui_watched` flag or whatever to extmark_set)

Reply:
I just found that this macro is not needed anymore :D